CefSharp.OffScreen/ChromiumWebBrowser.cs

```diff
@@ -663,6 +784,16 @@ bool IRenderWebBrowser.StartDragging(IDragData dragData, DragOperationsMask mask
         /// <param name="show">if set to <c>true</c> [show].</param>
         void IRenderWebBrowser.SetPopupIsOpen(bool show)
         {
+            if (!show && PopupImage != null)
```

Comment:
Preferable if the `true` condition comes first. In this case just set `PopupJustOpened = show` and perform your other checks as they are now, reduces code and branching.